srcpkgs/black/template

```diff
@@ -1,19 +1,20 @@
 # Template file for 'black'
 pkgname=black
-version=22.12.0
+version=23.1.0
 revision=1
 build_style=python3-pep517
 hostmakedepends="hatchling hatch-vcs"
-depends="python3-click python3-platformdirs python3-pathspec python3-mypy_extensions"
-checkdepends="${depends} python3-pytest-xdist python3-aiohttp python3-colorama python3-uvloop
+depends="python3-click python3-platformdirs python3-pathspec python3-packaging
```

Comment:
Does placing `python3-packaging` in `hostmakedepends` not work?

Reply:
No, packaging info required by the source https://github.com/psf/black/blob/23.1.0/src/black/files.py#L21